test/state/block.hpp

```diff
@@ -64,6 +66,10 @@ struct BlockInfo
 /// Computes the current blob gas price based on the excess blob gas.
 intx::uint256 compute_blob_gas_price(uint64_t excess_blob_gas) noexcept;
 
+/// Computes the current excess blob gas based on the parent.
```

Comment:
"on the parent"? Do you mean from the information of the parent block?

Reply:
yeah, mental shortcut